clang/lib/Driver/ToolChains/Linux.cpp

```diff
@@ -826,6 +826,10 @@ SanitizerMask Linux::getSupportedSanitizers() const {
   if (IsX86_64 || IsAArch64) {
     Res |= SanitizerKind::KernelHWAddress;
   }
+  if (IsX86_64) {
+    Res |= SanitizerKind::NumericalStability;
```

Comment:
drop braces for single-line simple statement

Reply:
This needs a `fsanitize-*.c` clang/test/Driver test

Reply:
Done